bin/decrypt_kernel

```diff
@@ -31,7 +31,7 @@ else
     warn_if_error
 fi
 
-lzss_offset=$(./get_lzss_section_offset.py "$kernelcache_decrypted")
+lzss_offset=$(python3 ./get_lzss_section_offset.py "$kernelcache_decrypted")
```

Comment:
Please also update the shebang line in `bin/get_lzss_section_offset.py` to `#!/usr/bin/env python3`.